include/groonga/portability.h

```diff
@@ -204,8 +204,10 @@
 # define grn_time_t __time64_t
 # define grn_time(dest_time) _time64((dest_time))
 # define grn_mktime(tm) _mktime64((tm))
+# define grn_timegm(tm) _mkgmtime64((tm))
 #else /* WIN32 */
 # define grn_time_t time_t
 # define grn_time(dest_time) time((dest_time))
 # define grn_mktime(tm) mktime((tm))
+# define grn_timegm(tm) timegm((tm))
```

Comment:
If we need to support environments that have no `timegm`, we should create our own `timegm` and use it in those environments.
I think we don't support those environments for now because the code below regard we always have `timegm` in the environments except Windows.

https://github.com/groonga/groonga/blob/master/plugins/functions/time.c#L33